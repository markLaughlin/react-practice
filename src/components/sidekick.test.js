import React from 'react';
import ReactDOM from 'react-dom';
import Comp from './sidekick';
import renderer from 'react-test-renderer';

describe('My component', () => {

    it('renders without crashing', () => {
        const div = document.createElement('div');
        ReactDOM.render(<Comp />, div);
        ReactDOM.unmountComponentAtNode(div);
      });
      
      it('renders the UI as expected', () => {
          const tree = renderer
            .create(<Comp/>)
            .toJSON();
          expect(tree).toMatchSnapshot();  
          });
          
});