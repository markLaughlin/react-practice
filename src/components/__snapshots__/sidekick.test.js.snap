// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`My component renders the UI as expected 1`] = `
<div>
  Here is an instance of a comp component. Aren't we having fun?
</div>
`;
